lib/ansible/plugins/callback/default.py

```diff
@@ -44,8 +45,19 @@ def __init__(self):
         self._last_task_banner = None
         self._last_task_name = None
         self._task_type_cache = {}
+        self._variable_manager = VariableManager()
+        self._extra_vars = self._variable_manager.extra_vars
         super(CallbackModule, self).__init__()
 
+    def get_option(self, option_name):
+
+        # Check extra_vars first
+        if option_name in self._extra_vars:
+            return self._extra_vars.get(option_name).lower() == 'true'
+
+        # Fallback to default functionality
+        return super(CallbackModule, self).get_option(option_name)
```

Comment:
In Python 3, there's a less verbose way:
```suggestion
        return super().get_option(option_name)
```

Reply:
Updated default.py